_posts/2023-03-11-release.md

```diff
@@ -0,0 +1,21 @@
+---
+title: Release 2023.10
+author: Priya
+date: 2023-03-11
+category: Jekyll
+layout: post
+---
+
+### Enhancements
+
+#### Revamped BigQuery Connector for Better User Experience
```

Comment:
#### Smart combine on Big Query is now available as an alpha feature (on request)

Reply:
done